convert2rhel/pkghandler.py

```diff
@@ -365,6 +369,21 @@ def preserve_only_rhel_kernel():
     return
 
 
+def install_gpg_keys():
+    loggerinst = logging.getLogger(__name__)
+    loggerinst.info("Importing Red Hat GPG keys ...")
+    gpg_path = os.path.realpath(os.path.join(os.path.dirname(__file__),
+                                "data/version-independent/gpg-keys/*"))
+    for gpg_key in glob.glob(gpg_path):
+        shutil.copy(gpg_key, _rpm_gpg_dir)
```

Comment:
Right now I'm not sure if copying the keys is a good idea, because the redhat-release package contains the keys as well and installs them to the same location. Let me check if it does not cause a conflict during the installation of redhat-release.

Reply:
Would you think it's better to just import the keys right from c2r instead of copying over and importing?

Reply:
Yeah, I think it would be better. Probably, when somebody uninstalls convert2rhel later and thus the keys won't be available, it should not matter because during the conversion the redhat-release package gets installed and it contains these keys.